composer.json

```diff
@@ -12,7 +12,8 @@
     "require": {
         "php": ">=7.2.5",
         "ext-json": "*",
-        "ext-sodium": "*"
+        "ext-sodium": "*",
```

Comment:
Don't we stop requiring `ext-sodium` since we're requiring the polyfill?

Reply:
Yep. Fixed in the next commit.